pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonControllerBuilder.java

```diff
@@ -299,6 +301,10 @@ protected abstract MiningCoordinator createMiningCoordinator(
 
   protected abstract ProtocolSchedule<C> createProtocolSchedule();
 
+  protected boolean validateContext(final ProtocolContext<C> context) {
+    return true;
```

Comment:
Remove the return value - nothing appears to use it.

Reply:
yeah .... oops .... was thinking that if the genesis was invalid, that may we'd stop execution ... but, think the log is plenty...